lib/recurly/subscription_list.php

```diff
@@ -8,8 +8,24 @@ public static function getActive($params = null, $client = null) {
   public static function getTrials($params = null, $client = null) {
     return Recurly_SubscriptionList::get(Recurly_Pager::_setState($params, 'in_trial'), $client);
   }
-  public static function getNonRenewing($params = null, $client = null) {
-    return Recurly_SubscriptionList::get(Recurly_Pager::_setState($params, 'non_renewing'), $client);
+  public static function getCanceled($params = null, $client = null) {
+    return Recurly_SubscriptionList::get(Recurly_Pager::_setState($params, 'canceled'), $client);
+  }
+
+  public static function getExpired($params = null, $client = null) {
+    return Recurly_SubscriptionList::get(Recurly_Pager::_setState($params, 'expired'), $client);
+  }
+  
+  public static function getFuture($params = null, $client = null) {
+    return Recurly_SubscriptionList::get(Recurly_Pager::_setState($params, 'future'), $client);
+  }
+  
+  public static function getLive($params = null, $client = null) {
+    return Recurly_SubscriptionList::get(Recurly_Pager::_setState($params, 'live'), $client);
+  }
+  
+  public static function getPastDue($params = null, $client = null) {
+    return Recurly_SubscriptionList::get(Recurly_Pager::_setState($params, 'past_due'), $client);
```

Comment:
Ah yeah good catch looks like you've got all of them. Nit pick but all your new methods are sorted, do you mind putting them above `getTrials` so it's all sorted?